cli/tests/it/script.rs

```diff
@@ -397,10 +397,10 @@ forgetest_async!(
             .run(ScriptOutcome::OkBroadcast)
             .assert_nonce_increment_addresses(vec![(
                 Address::from_str("0x90F79bf6EB2c4f870365E785982E1f101E93b906").unwrap(),
-                2,
+                1,
             )])
             .await
-            .assert_nonce_increment(vec![(0, 1)])
+            .assert_nonce_increment(vec![(0, 2)])
```

Comment:
Fixing the test here since this **should** be failing, and it did get [fixed](https://github.com/joshieDo/foundry/blob/d85c83a3f604431cb886c5aab048fc35686ab303/cli/src/cmd/forge/script/mod.rs#L352-L353) in this branch.